src/coreclr/src/jit/lower.cpp

```diff
@@ -2148,16 +2147,11 @@ void Lowering::RehomeArgForFastTailCall(unsigned int lclNum,
 }
 
 //------------------------------------------------------------------------
-// LowerTailCallViaHelper: lower a call via the tailcall helper. Morph
-// has already inserted tailcall helper special arguments. This function
-// inserts actual data for some placeholders.
+// LowerTailCallViaJitHelper: lower a call via the tailcall JIT helper. Morph
+// has already inserted tailcall helper special arguments. This function inserts
+// actual data for some placeholders. This function is only used on x86.
 //
-// For ARM32, AMD64, lower
-//      tail.call(void* copyRoutine, void* dummyArg, ...)
-// as
-//      Jit_TailCall(void* copyRoutine, void* callTarget, ...)
-//
-// For x86, lower
+// Lower
```

Comment:
Typo? Suggest removing.

Reply:
This comment is
```
// Lower
//   x
// into
//   y
```
I rewrote it because this now only deals with the x86 jit helper.